subiquity/models/mirror.py

```diff
@@ -51,12 +51,20 @@ def __init__(self):
         self.config = copy.deepcopy(DEFAULT)
         self.architecture = get_architecture()
         self.default_mirror = self.get_mirror()
+        self.disable_components = set()
 
-    def is_default(self):
+    def get_config(self):
+        # not the same as render() - this receives special processing rather
```

Comment:
This comment is out of date now given that render() returns {} now?

Reply:
done